src/__tests__/utils/password.test.ts

```diff
@@ -0,0 +1,18 @@
+import { checkHash, hashFunction } from '../../utils/password';
+
+describe('testing the hashFunction', () => {
+  it('should compare the password hashed and without', () => {
+    expect(hashFunction('password')).not.toBe('password');
+  });
+
+  it('should compare the length of 2 hashed passwords', async () => {
+    const hashed1 = await hashFunction('password1');
+    const hashed2 = await hashFunction('pw2');
+    expect(hashed1.length).toBe(hashed2.length);
+  });
+});
+describe('testing the hashFunction', () => {
+  it('should return the same results', async () => {
```

Comment:
this feels like a bad explanation of what this does, it's more so testing that the same secret always creates the same hashed value